test/browser_tests/page_objects/page_wagtail_templates.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+var _getQAElement = require( '../util/qa-element' ).get;
+
+function LandingPage() {
+
+  this.get = function() {
+    var baseUrl = '/landing-page/';
+    browser.get( baseUrl );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+}
+
+function SubLandingPage() {
+
+  this.get = function() {
+    var baseUrl = '/sublanding-page/';
+    browser.get( baseUrl );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+}
+
+function BrowsePage() {
+
+  this.get = function() {
+    var baseUrl = '/browse-page/';
+    browser.get( baseUrl );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+}
+
+function BrowseFilterablePage() {
+
+  this.get = function() {
+    var baseUrl = '/browse-filterable-page/';
+    browser.get( baseUrl );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+}
+
+function SublandingFilterablePage() {
+
+  this.get = function() {
+    var baseUrl = '/sublanding-filterable-page/';
+    browser.get( baseUrl );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+}
+
+function EventArchivePage() {
+
+  this.get = function() {
+    var baseUrl = '/event-archive-page/';
+    browser.get( baseUrl );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+}
+
+function DocumentDetailPage() {
+
+  this.get = function() {
+    var baseUrl = '/browse-filterable-page/document-detail';
```

Comment:
@richaagarwal when creating the pages make sure the docDetail/Learn/event live under a browse-filterable-page